src/patternfly/components/Alert/alert.scss

```
@@ -21,6 +21,7 @@
   // Toggle
   --#{$alert}__toggle--MarginBlockStart: calc(-1 * var(--pf-t--global--spacer--control--vertical--default));
   --#{$alert}__toggle--MarginBlockEnd: calc(-1 * var(--pf-t--global--spacer--control--vertical--default));
+  --#{$alert}__toggle--MarginInlineStart: calc(-1 * var(--pf-t--global--spacer--sm));
```

any reason not to use the semantic spacer here? That way the style continues to work if we update horizontal padding for buttons.

```suggestion
  --#{$alert}__toggle--MarginInlineStart: calc(-1 * var(--pf-t--global--spacer--action--horizontal--plain--default));
```